code/modules/mining/mine_turfs.dm

```diff
@@ -278,25 +415,17 @@
 					M.Stun(5)
 			M.apply_effect(25, IRRADIATE)
 
-
-	var/list/step_overlays = list("n" = NORTH, "s" = SOUTH, "e" = EAST, "w" = WEST)
-
-	//Add some rubble,  you did just clear out a big chunk of rock.
-
-	var/turf/simulated/floor/asteroid/N = ChangeTurf(mined_turf)
-
 	// Kill and update the space overlays around us.
+	var/list/step_overlays = list("n" = NORTH, "s" = SOUTH, "e" = EAST, "w" = WEST)
 	for(var/direction in step_overlays)
 		var/turf/space/T = get_step(src, step_overlays[direction])
 		if(istype(T))
 			T.overlays.Cut()
 			for(var/next_direction in step_overlays)
 				if(istype(get_step(T, step_overlays[next_direction]),/turf/simulated/mineral))
-					T.overlays += image('icons/turf/walls.dmi', "rock_side_[next_direction]")
+					T.overlays += image('icons/turf/walls.dmi', "rock_side", dir = step_overlays[next_direction])
```

Comment:
It was originally used as part of "rock_side", whoever fixed it to be directional didn't update the code thoroughly.